pkg/snapstore/snapshot.go

```diff
@@ -23,14 +23,25 @@ import (
 
 // GenerateSnapshotName preapres the snapshot name from metadata
 func (s *Snapshot) GenerateSnapshotName() {
-	s.SnapPath = fmt.Sprintf("%s-%08d-%08d-%08d", s.Kind, s.StartRevision, s.LastRevision, s.CreatedOn.Unix())
+	s.SnapName = fmt.Sprintf("%s-%08d-%08d-%d", s.Kind, s.StartRevision, s.LastRevision, s.CreatedOn.Unix())
 }
 
-// ParseSnapshot parse <snapName> to create snapshot structure
-func ParseSnapshot(snapName string) (*Snapshot, error) {
+// GenerateSnapshotDirectory prepares the snapshot directory name from metadata
+func (s *Snapshot) GenerateSnapshotDirectory() {
+	s.SnapDir = fmt.Sprintf("%s-%d", SnapshotKindFull, s.CreatedOn.Unix())
```

Comment:
Instead of dir name being something like 'Full-34234234' in the directory name, is it not better to have it like 'Backup-34234234'. We shall have the full backups and the incrementals inside this directory.

Reply:
Make sense. I'll change that.